doc/changelog/03-notations/11986-float-low-level-printing.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  Add flag ``Printing Float`` to print primitive floats as hexadecimal
```

Comment:
Sorry for seeing this too late but you could have linked directly to the doc like this:
```suggestion
  Add flag :flag:`Printing Float` to print primitive floats as hexadecimal
```

Reply:
Thanks for the information. Maybe not worth a new PR?

Reply:
Yeah, maybe not.